modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/edit_configuration.jsp

```diff
@@ -31,7 +31,7 @@ String ddmFormHTML = (String)request.getAttribute(ConfigurationAdminWebKeys.CONF
 String category = configurationModel.getCategory();
 
 portletDisplay.setShowBackIcon(true);
-portletDisplay.setURLBack(portletURL.toString());
+portletDisplay.setURLBack(redirect);
```

Comment:
This is fine as long as we default "redirect" to portletURL.toString() if it's not been provided.